databricks/koalas/tests/test_utils.py

```diff
@@ -35,6 +35,16 @@ def to_html(self, max_rows=None, unsupported_param=None):
         }, index=[0, 1, 3])
         validate_arguments_and_invoke_function(pdf, self.to_html, pd.DataFrame.to_html, args)
 
+    def to_clipboard(self, sep=',', **kwargs):
```

Comment:
why is this test case here? this file is for testing functionalities in utils.py

Reply:
This test case is for testing `**kwargs` in `validate_arguments_and_invoke_function()`

> Improve validate_arguments_and_invoke_function() when receiving argument **kwargs
> - actual args : {'kwargs': {'index': False}, 'sep': ',', 'excel': True}
> - expected args : {'sep': ',', 'excel': True, 'index': False}